WordPressVIPMinimum/Tests/Security/ProperEscapingFunctionUnitTest.inc

```diff
@@ -30,12 +30,44 @@ echo "<a title='" . esc_html( $some_var ) . "'></a>"; // NOK.
 
 echo '<media:content url="' . esc_url( $post_image ) . '" medium="image">'; // OK.
 
-echo '<media:content url="' . esc_attr( $post_image ) . '" medium="image">'; // NOK.
+echo '<media:content url="' . esc_attr( $post_image ) . '" medium="image">'; // Error.
 
 echo 'data-param-url="' . esc_url( $share_url ) . '"'; // OK.
 
-echo 'data-param-url="' . esc_html( $share_url ) . '"'; // NOK.
+echo 'data-param-url="' . esc_html( $share_url ) . '"'; // Error.
 
 ?>
 
 <form method="post" action="<?php echo esc_html(admin_url('admin.php?page='.$base_name.'&amp;mode=logs&amp;id='.$poll_id)); ?>">
+
+?>
```

Comment:
You're not in PHP at this point, so this `?>` is superfluous (and can cause the test code to be tokenized incorrectly).